### PR TITLE
Update new readme to match radix api

### PR DESCRIPTION
**Pull request by Alek99**
The api for input is not the best right now but this is how it works with our current functionality

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,41 +90,51 @@ class State(rx.State):
...
+                    ),
+                ),
             ),
-            padding="2em",
-            shadow="lg",
-            border_radius="lg",
+            width="25em",
+            padding="1em",
+            spacing="2",
+            align_items="center",
```
Comment (picklelo) [inline README.md:126]: This should be `align` now

File: README.md [hunk 2]
```diff
@@ -88,41 +90,51 @@ class State(rx.State):
...
-                "Generate Image",
-                on_click=State.get_image,
-                is_loading=State.processing,
+            rx.heading("DALL-E", font_size="1.5em"),
+            rx.radix.text_field.root(
+                rx.input(
+                    placeholder="Enter a prompt..",
+                    on_blur=State.set_prompt,
+                    width="100%",
+                ),
```
Comment (masenf) [inline README.md:109]: why is this wrapped in a root? maybe we can just set the width to some relative unit, like `width="25em"`?

Comment (Alek99) [reply README.md:109]: It has to because there is no way to affect the percentage width otherwise

Comment (Alek99) [reply README.md:109]: Ok I can do that for now

File: README.md [hunk 3]
```diff
@@ -72,10 +72,12 @@ Here is the complete code to create this. This is all done in one Python file!
 import reflex as rx
 import openai
 
-openai.api_key = "YOUR_API_KEY"
+openai_client =  openai.OpenAI(api_key="YOUR_API_KEY")
```
Comment (picklelo) [inline README.md:75]: I think in the new api you don't need to pass in the key, it just reads it from the env var. (Maybe we can just put a comment telling them to make sure its set)

File: README.md [hunk 4]
```diff
@@ -88,41 +90,50 @@ class State(rx.State):
...
+                ),
             ),
-            padding="2em",
-            shadow="lg",
-            border_radius="lg",
+            width="25em",
+            padding="1em",
+            spacing="2",
+            align_items="center",
+            border_radius="10px",
```
Comment (picklelo) [inline README.md:127]: Potentially confusing to newcomers that we're mixing px, em, and % all together for. maybe we can unify for the intro app

**Review by picklelo — COMMENTED** [review]
looks good, can we run `black` on the code example, then we can merge this

**Review by picklelo — APPROVED** [review]

**Pull request merged**